### DISCUSSION TITLE
[Script request] Immich

### DISCUSSION THREAD

**Discussion by StefanKnol** [community-scripts/ProxmoxVE#818] [Request script] [answered]
### Application Name

Immich

### Website

https://immich.app/

### Description

From their site:
Self-hosted photo and video management solution

Easily back up, organize, and manage your photos on your own server. Immich helps you browse, search and organize your photos and videos with ease, without sacrificing your privacy.

can already be installed in docker, would love to skip that step with an lxc

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment, marked as answer]
Use search function and you see its rejected 

Use dockge
